core/common/src/main/java/alluxio/util/ShellUtils.java

```diff
@@ -234,14 +341,131 @@ public String toString() {
   }
 
   /**
-   * Static method to execute a shell command.
+   * Object representation of a command execution.
+   */
+  public static class CommandReturn {
```

Comment:
this might make sense as a separate class, or a part of `Command` (which should be a separate class)